app/assets/stylesheets/_colors.scss

```diff
@@ -0,0 +1,29 @@
+$smoke: #08090a;
```

Comment:
I'm undecided, but I'm wondering if saying `$smoke` or `$smoke-100` might be preferable here. Since I see the other `smoke` options with their number, seeing it as simply `$smoke` feels a bit incomplete.

Reply:
Agree. I'll update to `$smoke-100`.